Marlin/src/module/tool_change.cpp

```diff
@@ -784,20 +780,6 @@ void tool_change(const uint8_t tmp_extruder, const float fr_mm_s/*=0.0*/, bool n
         lower_nozzle(active_extruder);
       #endif
 
-      #if ENABLED(TOOLCHANGE_FILAMENT_SWAP) && ADVANCED_PAUSE_RESUME_PRIME != 0
-        if (should_swap && !too_cold) {
-          const float resume_eaxis = current_position[E_AXIS];
-          #if ENABLED(ADVANCED_PAUSE_FEATURE)
-            do_pause_e_move(toolchange_settings.swap_length, toolchange_settings.prime_speed);
-          #else
-            current_position[E_AXIS] += (ADVANCED_PAUSE_RESUME_PRIME) / planner.e_factor[active_extruder];
-            planner.buffer_line(current_position, ADVANCED_PAUSE_PURGE_FEEDRATE, active_extruder);
-          #endif
-          planner.synchronize();
-          planner.set_e_position_mm((destination[E_AXIS] = current_position[E_AXIS] = resume_eaxis));
-        }
-      #endif
-
```

Comment:
This change and the one above are pretty drastic admissions of mass-confusion. Why are these blocks no longer needed, and what made us think they were needed before? What code is taking the place of this application of tool-change extruder priming?

Reply:
From what I can gather initially the code for the prime/retract and extra prime were separate then they got merged and the second instance never removed. Most likely some kind of merge conflict that got overlooked as we crashed between each other and marcio at one point while working on this last.

Reply:
Perfecto!